bolt/test/lit.cfg.py

```diff
@@ -92,10 +92,22 @@
 
 tool_dirs = [config.llvm_tools_dir, config.test_source_root]
 
+llvm_bolt_args = []
+
+if config.libbolt_rt_instr:
+    llvm_bolt_args.append(f"--runtime-instrumentation-lib={config.libbolt_rt_instr}")
```

Comment:
In --runtime-instrumentation-lib description, it says:

"specify file name of the runtime instrumentation library"

But here we provide a full path to the library. It should fail to work, as #99806 implies.

However, the reason it succeeds (at least on my system) is because we're adding here the getLibPathByInstalled() function that is currently broken, CMAKE_INSTALL_LIBDIR resolves to a single folder name (lib64), that then gets converted into an empty string via the call to root_path, and then we finally append the contents of --runtime-instrumentation-lib, which succeeds to find the file because it is already a full path.

I think if the intent is to use --runtime-instrumentation-lib as a flag that contains the full path, we should at least update the description of that and properly support it in getLibPath() (instead of having it work by chance).

Reply:
> CMAKE_INSTALL_LIBDIR resolves to a single folder

Ah, I kinda knew this was an issue but couldn't remember why until I looked at the [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) docs. I need to use the full variant and this should be fixed.

> update the description of that and properly support it in getLibPath() (instead of having it work by chance).

Yeah, that sounds good. I don't really understand why it's a filename only when there's proper use cases like with Nix (and probably many others) where you would want to specify a path.

Reply:
Alright, I've updated the description and switched to the full path now.